morpheus/utils/controllers/rss_controller.py

```diff
@@ -36,13 +40,13 @@ def __init__(self, feed_input: str, batch_size: int = 128):
         self._batch_size = batch_size
         self._previous_entires = set()  # Stores the IDs of previous entries to prevent the processing of duplicates.
         # If feed_input is URL. Runs indefinitely
-        self._run_indefinitely = True if RSSController.is_url(feed_input) else False
+        self._run_indefinitely = RSSController.is_url(feed_input)
 
     @property
     def run_indefinitely(self):
         return self._run_indefinitely
 
-    def parse_feed(self):
+    def parse_feed(self) -> typing.List:
```

Comment:
You can just use `list` now. 
```suggestion
    def parse_feed(self) -> list[dict]:
```